common/libs/validators.js

```diff
@@ -143,5 +143,7 @@ export function isValidRawTx(rawTx: RawTransaction): boolean {
 // Full length deterministic wallet paths from BIP32
 // https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki
 export function isValidPath(dPath: string) {
-  return dPath.split("'/").length === 4;
+  //TODO: use a regex to detect proper paths
+  const len = dPath.split("'/").length;
+  return len === 3 || len === 4;
```

Comment:
Ledger uses `m/44'/60'/0'` for some reason:

https://ethereum.stackexchange.com/questions/19055/what-is-the-difference-between-m-44-60-0-0-and-m-44-60-0/19061

Reply:
Oh wow, BIP44 is way more readable than BIP32. Would have saved me a lot of headaches: https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki

Would you mind changing the comment to reflect that BIP 44 is really where we're determining that 4 is the amount, but that ledger is the exception for 3?

Reply:
yep, will do :+1: